server/core/terraform/terraform_client_test.go

```diff
@@ -507,3 +507,298 @@ terraform {
 		runDetectVersionTestCase(t, name+": Downloads Disabled", testCase, false)
 	}
 }
+
+func TestDefaultProjectCommandBuilder_TerraformVersionV1_0_0(t *testing.T) {
```

Comment:
Why duplicate this test instead of modifying the existing test?

Reply:
I do realize it's not the best way to do it but I just went for the quick and dirty way because the original test does not have an easy way to add more test cases

Reply:
I figured lol.

Thank you for contributing this fix but could you please modify the existing test to support multiple cases for `~>` to reduce the test code to maintain?

Reply:
Yeah sure, might take some time but I'll try to do it asap